CalibTracker/SiStripChannelGain/python/ntuple_cff.py

```diff
@@ -12,14 +23,53 @@
                                      'keep *_shallowTracks_tracketa_*',
                                      'keep *_shallowTracks_trackphi_*',
                                      'keep *_shallowTracks_trackhitsvalid_*',
+                                     'keep *_shallowTracks_trackalgo_*',
                                      'keep *_shallowGainCalibration_*_*'
```

Comment:
@mmusich

We need the track algorithm iteration used to reconstruct the track because the track good for the calibration are selected to be track_ago < 8. 
> On 09 Mar 2016, at 13:29, Marco Musich <notifications@github.com> wrote:
> 
> In CalibTracker/SiStripChannelGain/python/ntuple_cff.py <https://github.com/cms-sw/cmssw/pull/13609#discussion_r55511142>:
> 
> > @@ -12,14 +23,53 @@
> >                                       'keep *_shallowTracks_tracketa_*',
> >                                       'keep *_shallowTracks_trackphi_*',
> >                                       'keep *_shallowTracks_trackhitsvalid_*',
> > +                                     'keep *_shallowTracks_trackalgo_*',
> >                                       'keep *_shallowGainCalibration_*_*'
> @dimattia <https://github.com/dimattia> can you clarify with this new event product is needed?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cms-sw/cmssw/pull/13609/files#r55511142>.
>